src/test/java/com/github/bbooong/bangumall/stock/domain/StocksTest.java

```diff
@@ -99,8 +83,8 @@ class Context_With_QuantityLessThanOrEqualToTotalQuantity {
             void it_decreases_quantity() {
                 stocks.decreaseQuantity(15);
                 assertAll(
-                        () -> assertThat(olderStock.getQuantity()).isEqualTo(0),
-                        () -> assertThat(newerStock.getQuantity()).isEqualTo(15));
+                        () -> assertThat(olderStock.getQuantity().getValue()).isEqualTo(0),
+                        () -> assertThat(newerStock.getQuantity().getValue()).isEqualTo(15));
```

Comment:
```suggestion
                        () -> assertThat(olderStock.getQuantity()).isEqualTo(new Quantity(0)),
                        () -> assertThat(newerStock.getQuantity()).isEqualTo(new Quantity(15));
```

Equals 재정의 했으니 위 코드가 적절해 보여요